<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookFormUser</class>
 <widget class="QDialog" name="AddBookFormUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="iSBNLabel">
         <property name="text">
          <string>ISBN</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="iSBNLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="titleLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="authorLabel">
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="authorLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="editionLabel">
         <property name="text">
          <string>Edition</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editionLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="publisherLabel">
         <property name="text">
          <string>Publisher</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="publisherLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="formatLabel">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="formatLineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPlainTextEdit" name="DescriptionTextEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="publishingDateLabel">
         <property name="text">
          <string>Published</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDateEdit" name="publishingDateDateEdit">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="date">
          <date>
           <year>1753</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
